README.md

```diff
@@ -1,39 +1,20 @@
 # Protelis demo application
 
-This project demonstrates a minimal application making use of Protelis.
+The goal of this project is project is to show some basic implementations of [Protelis](https://github.com/Protelis/Protelis) using both Java and Kotlin.
 
-Protelis requires Java 8+.  This project has been set up to run in Eclipse with Maven, and should be able to be 
-directly imported and executed if you have these set up.
+Protelis requires Java 8+.
```

Comment:
Add comment on the structure:
* gradle
* how to import in Eclipse and Intellij